oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentStoreStats.java

```diff
@@ -277,12 +277,22 @@ public void doneCreateOrUpdate(long timeTakenNanos, Collection<? extends Documen
     public void doneFindAndModify(long timeTakenNanos, Collection<? extends Document> collection, String key, boolean newEntry,
                                   boolean success, int retryCount) {
 
-        modifyMetricUpdater.update(collection, retryCount, timeTakenNanos, success, newEntry, c -> c == NODES, getStatsConsumer(),
+        modifyMetricUpdater.update(collection, retryCount, timeTakenNanos, success, newEntry, of(key), isNodesCollectionUpdated(), getStatsConsumer(),
                 getStatsConsumer(), MeterStats::mark, MeterStats::mark);
 
         perfLog(perfLog, PERF_LOG_THRESHOLD, timeTakenNanos, "findAndModify [{}]", key);
     }
 
+    @Override
+    public void doneFindAndModify(long timeTakenNanos, Collection<? extends Document> collection, List<String> ids, boolean newEntry,
```

Comment:
Can you please add tests for this method in `DocumentStoreStatsIT` and `DocumentStoreStatsTest`? Similar to method variant with single key/id.